modules/data.atmosphere/R/NARR_csv.R

```diff
@@ -0,0 +1,315 @@
+#' Download NARR time series as CSV
+#'
+#' @param outfolder Target directory for storing output
+#' @param start_date Start date for met data
+#' @param end_date End date for met data
+#' @param lat.in Site latitude coordinate
+#' @param lon.in Site longitude coordinate
+#' @param overwrite Overwrite existing files?  Default=FALSE
+#' @param verbose Turn on verbose output? Default=FALSE
+#' 
+#' @examples
+#' download.NARR_csv(tempdir(), "2001-01-01", "2001-01-12", 43.372, -89.907)
+#' 
+#' @export
+#'
+#' @author Alexey Shiklomanov
+download.NARR_csv <- function(outfolder, start_date, end_date, lat.in, lon.in, overwrite = FALSE, verbose = FALSE, ...) {
```

Comment:
Is there a particular reason you'd want to download a CSV (other than it is useful now and then). If want to plug into met.process, this adds the extra step of then running met2CF.csv, which then needs the format defined in Formats table. I

Reply:
Nope. I'll change it accordingly.